rero_ils/modules/items/api/record.py

```diff
@@ -353,6 +353,17 @@ def holding_circulation_category_pid(self):
                     self.holding_pid).circulation_category_pid
         return circulation_category_pid
 
+    def item_inherited_first_call_number(self):
+        """Return holdings first call number if item 1st call# is missing."""
+        from ...holdings.api import Holding
+        holding = Holding.get_record_by_pid(self.holding_pid)
+        if (
+            self.get('type') == 'issue' and
+            not self.get('call_number') and
+            holding.get('call_number')
+        ):
+            return holding.get('call_number')
```

Comment:
In this case, I suggest to split the `if` statement in 2 to avoid loading the holding if it's not necessary (best perf).
```
    if (self.get('type') == 'issue' and not self.get('call_number')):
        return Holding.get_record_by_pid(self.holding_pid).get('call_number')
```
the `if holding.get('call_number'):` can be skipped because in both case the function will return 'None' (a function without specific return statement return None by default)

More I read this function, more I think it could be a `call_number item @properety`. If it could then previous change could be removed.
```
# inherit holdings first call# for issues with no 1st call#.
inherited_call_number = item.item_inherited_first_call_number()
if inherited_call_number:
    item['call_number'] = inherited_call_number
```
==> a call number for an issue without specific call_number is :
  - None if parent holding doesn't have a call_number
  - The parent holding call_number